src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/UnSetAttributeCommand.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.References
+{
+    internal class UnSetAttributeCommand : IProjectSystemUpdateReferenceOperation
```

Comment:
>UnSet [](http://example.com/codeflow?start=19&length=5)

nit: as far as naming, isn't "Unset" one word? if so, the uppercase S in UnSet seems a bit off.

Reply:
done